sgxs-loaders/src/sgx_enclave_common/mod.rs

```diff
@@ -244,13 +246,26 @@ impl EnclaveLoad for InnerLibrary {
             ) {
                 return Err(Error::Init(error.into()));
             }
-
+            #[cfg(unix)]
             libc::mprotect(
                 mapping.base as _,
                 mapping.size as _,
                 libc::PROT_READ | libc::PROT_WRITE | libc::PROT_EXEC,
             );
```

Comment:
Not sure why this wasn't here yet already, please add error handling similar to the Windows case.